src/main.py

```diff
@@ -2728,6 +2738,7 @@ def post_all(self, o):
             asm3.configuration.cset(o.dbo, "%s_QuicklinksID" % o.user, "")
         else:
             asm3.configuration.cset(o.dbo, "%s_QuicklinksID" % o.user, quicklinks)
+        asm3.configuration.cset(o.dbo, "%s_FavouriteReportsID" % o.user, favouritereports)
```

Comment:
here's where you do ",".join(quickreportscfg)

Reply:
Already done :)